core/jazz_metrics/components/validation.js

```diff
@@ -41,6 +43,44 @@ var validateGeneralFields = (input) => {
   });
 }
 
+function validateDate(startDate, endDate) {
+  return new Promise((resolve, reject) => {
+    if(!moment(startDate).isValid()) {
+      reject({
+        result: "inputError",
+        message: `Start date time is not in valid format`
+      })
+    }
+    if(!moment(endDate).isValid()) {
+      reject({
+        result: "inputError",
+        message: `End date time is not in valid format`
+      })
+    }
```

Comment:
Can we add a check for startDate < endDate?

Reply:
that check is already there in validateMetricsInput function

Reply:
Cool, missed it I guess.